### PR TITLE
[DOCS-7162] Add column width section

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/using/index.md [hunk 1]
```diff
@@ -205,6 +205,21 @@ The columns that are visible can be adjusted.
 
 ![Select columns]({% link process-automation/images/select-columns.png %})
 
+### Column width
+
+You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you have changed one column, all the columns with the same name are also adjusted.
```
Comment (rafalszmit) [inline process-automation/latest/using/index.md:210]: You can add that the column size is saved in preferences. If user will log out and in, it should stay the same.

**Review by rafalszmit — APPROVED** [review]

File: process-automation/latest/using/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/using/index.md]: Done.

File: process-automation/latest/using/index.md [hunk 2]
```diff
@@ -205,6 +205,21 @@ The columns that are visible can be adjusted.
 
 ![Select columns]({% link process-automation/images/select-columns.png %})
 
+### Column width
+
+You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you have changed one column, all the columns with the same name are also adjusted. If you log out and then log back in again the new column widths will be preserved.
+
+This feature is enabled by default for the **Process Management** section. If you want to adjust the column widths in the same way for the **Personal Files** or **File Libraries** sections of the Digital Workspace, you must turn it on by editing the `libs/content-ee/process-services-cloud-extension/assets/process-services-cloud.extension.json` file and adding the following:
+
+```JSON
```
Comment (anxumalo) [inline process-automation/latest/using/index.md:214]: Prefer lowercase markers:
```suggestion
```json
```

File: process-automation/latest/using/index.md [hunk 3]
```diff
@@ -205,6 +205,21 @@ The columns that are visible can be adjusted.
 
 ![Select columns]({% link process-automation/images/select-columns.png %})
 
+### Column width
+
+You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you have changed one column, all the columns with the same name are also adjusted. If you log out and then log back in again the new column widths will be preserved.
```
Comment (anxumalo) [inline process-automation/latest/using/index.md:210]: Rewrite using present tense where possible, for example:
```suggestion
You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you change one column, all the columns with the same name are also adjusted. If you log out and then log back in again, the new column widths is preserved.
```

**Review by anxumalo — CHANGES_REQUESTED** [review]
Minor fixes to consider

**Review by anxumalo — APPROVED** [review]

**Pull request merged**